client/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
```

Comment:
Remove this "use client"

Footer in server components it's not necessary always to use "use client" only use in client component.